src/plugins/intel_gpu/src/kernel_selector/kernel_selector_common.h

```diff
@@ -114,6 +114,16 @@ struct KernelData {
 
     bool can_reuse_memory = true;
 
+    template <typename op_params>
+    inline static bool SkipKernelExecution(const op_params& params, size_t kernel_id = 0) {
+        for (const auto& input : params.inputs) {
+            if (input.LogicalSize() == 0) {
```

Comment:
why not seeing the output? 
Even though there is inputs, we can skip the kernel if there is no output to be generated by the kernel.

Reply:
Added check for outputs